src/ControlSystem/SettleToConstant.hpp

```diff
@@ -0,0 +1,62 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <vector>
+
+#include "ControlSystem/FunctionOfTime.hpp"
+#include "DataStructures/DataVector.hpp"
+
+/// \ingroup ControlSystemGroup
+/// Contains functions of time to support the dual frame system.
+namespace FunctionsOfTime {
+/// \ingroup ControlSystemGroup
+/// Sets f(t) and derivatives via exponentials, such that f(t)
+/// approaches a constant in time.
+/// Function is f(t) = A + (B+C(t-t0)) exp(-(t-t0)/tau)
+/// where A,B,C,t0,tau are constants.
+class SettleToConstant : public FunctionOfTime {
+ public:
+  SettleToConstant(std::array<DataVector, 3> initial_func_and_derivs, double t0,
+                   double decay_time) noexcept;
+
+  ~SettleToConstant() override = default;
+  SettleToConstant(SettleToConstant&&) noexcept = default;
+  SettleToConstant& operator=(SettleToConstant&&) noexcept = default;
+  SettleToConstant(const SettleToConstant&) = delete;
+  SettleToConstant& operator=(const SettleToConstant&) = delete;
+
+  /// Returns the function at an arbitrary time `t`.
+  std::array<DataVector, 1> func(double t) const noexcept override {
+    return func_and_derivs<0>(t);
+  }
+  /// Returns the function and its first derivative at an arbitrary time `t`.
+  std::array<DataVector, 2> func_and_deriv(double t) const noexcept override {
+    return func_and_derivs<1>(t);
+  }
+  /// Returns the function and the first two derivatives at an arbitrary time
+  /// `t`.
+  std::array<DataVector, 3> func_and_2_derivs(double t) const
+      noexcept override {
+    return func_and_derivs<2>(t);
+  }
+
+  /// Returns the domain of validity of the function.
+  std::array<double, 2> time_bounds() const noexcept override {
+    return {{match_time_, std::numeric_limits<double>::max()}};
+  }
+
+ private:
+  /// Returns the function and 2 derivatives at an arbitrary time `t`.
+  /// The function has multiple components.
+  template <size_t MaxDerivReturned = 2>
+  std::array<DataVector, MaxDerivReturned + 1> func_and_derivs(double t) const
+      noexcept;
+
+  DataVector coef_a_, coef_b_, coef_c_;
```

Comment:
`coef_a_` is technically the asymptotic constant 'target' value, however {A,B,C} are all
computed from the specified `initial_func_and_derivs`. Since `coef_a_` is not a specifiable 
target, but will depend on the function at the matching time, it could be confusing if 
one of the coefficients does not follow the same naming convention as the others.